examples/csi_driver_install_vars.yaml

```diff
@@ -3,16 +3,19 @@ service_instance_id: "bc895dc9-f626-41c0-bfd3-94a0d8942a5f"  #IBM Cloud PowerVS
 region: "mon"                                                #IBM Cloud PowerVS service instance region
 zone: "mon01"                                                #IBM Cloud PowerVS service instance zone
 
+resource_type: "stable"                                      #CSI driver resource type.
+resource_version: "v0.1.0"                                   #CSI driver resource version.
```

Comment:
Any specific reason for naming these variables as `resource_*`? if someone looks at these without the explanation its very difficult to find the relevance!

may be `csi_driver_type` and `csi_driver_version` make more sense here, wdys @Madhan-SWE

Reply:
Fixed.